src/main/java/uk/gov/hmcts/darts/authorisation/annotation/Authorisation.java

```diff
@@ -18,4 +18,6 @@
 
     SecurityRoleEnum[] securityRoles();
 
+    SecurityRoleEnum[] globalAccessSecurityRoles();
```

Comment:
Think this should probably be `default {};`
So that we don't necessarily need to add this parameter, if not needed?